### ISSUE TITLE
Role and Label missing in Home screen

### ISSUE THREAD

**Issue by rgarg24** [percussion/percussioncms#840]
**Describe the bug**
Role and Label missing for various divs which are acting like buttons at Home screen.

**To Reproduce**
1. Go to Home screen "http://localhost:9992/cm/app/?view=home&pathType=unknown"
2. You will see various buttons like "Add New", 'Search", "Browse Library".
3. When we inspect them, we will see they are actually divs which are acting like buttons and they don't have any role or label.

**Expected behavior**
Role and Label should be present for divs or anchor tags which acts like buttons.

**Additional context**
Navigate all the options available for similar issue and fix.
